src/Core/Managed/Shared/ActivityExtensions.cs

```diff
@@ -12,30 +14,25 @@ internal static class ActivityExtensions
         private static bool isEnabled = false;
 
         /// <summary>
-        /// Checks that Activity API could be called (DiagnosticSource DLL is loaded).
```

Comment:
unfortunately previous version did not work. I assume runtime tries to resolve Activity and crashed when entering StartOperation method.
I'm working on the standalone test now